network.js

```diff
@@ -18,7 +18,8 @@ function blockFrostAmountToValue(obj) {
         if (item.unit == "lovelace") {
             value = value.add(new Value(qty));
         } else {
-            let mph = MintingPolicyHash.fromHex(item.unit);
+            let policyID = `${item.unit}`.substring(0, 56);
```

Comment:
if `item.unit` is a string, would it be better to write `item.unit.substring(0, 56)` directly?

Reply:
My intention was to be overly cautious given the string data type returned by `item.unit` is not enforced by a type checker. Not opposed to calling `item.unit.substring` for simplicity though, so went ahead and made the suggested change 👍